create_docker_certs.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
```

Comment:
This script will generate in and remove files for the cert and intermediate files in the folder where this script runs.
I'd create a temp folder with `mktemp -d`, and then place all the generated files into it. If generation all succeeded, then move them all to the destinations.